README.md

```diff
@@ -0,0 +1,215 @@
+# Docker image for Petals Deployer
+
+This image allows to configure and run a Petals CLI instance to deploy artifacts onto a running Petals ESB instance.
+
+Connection parameters and artifact list must be set at launch.
+Artifacts can be deployed by 
+* mounting a docker volume containnig artifacts zipfiles and giving files to deploy.
+* hosting artifacts on a nexus and giving artifacts url.
+>Both method can be used at once but artifacts as parameters will always be deployed first.
+
+
+## Launch a container from this image
+
+```properties
+# Download the image
+docker pull petalslink/petals-deployer:latest
+
+# Start in detached mode (with default config)
+docker run -d -p 80:80 --name petals-deployer petalslink/petals-deployer:latest
+
+# Start example #1
+docker run \
+	-d -p 80:80 --name petals-deployer \
```

Comment:
The port 80 is necessary only when there are local zipped files.
Otherwise, it is not used.

Reply:
This one uses local files, I changed it for the others.